examples/color/colormap_reference.py

```diff
@@ -5,17 +5,13 @@
 
 Reference for colormaps included with Matplotlib.
 
-A reversed version of each of these colormaps is available by appending
-``_r`` to the name, e.g., ``viridis_r``.
```

Comment:
I'd keep this, and reference below with an anchor link....  Otherwise this looks great